File: examples-sdk/kotlin/403-agent-code-file-reviewer/main.kt
// 403-agent-code-file-reviewer — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 403-agent-code-file-reviewer/main.kt --release

fun main() {
    val server = VilServer("code-file-reviewer-agent", 3122)
    val code_review_agent = ServiceProcess("code-review-agent")
    code_review_agent.endpoint("POST", "/code-review", "code_review_handler")
    server.service(code_review_agent)
    server.compile()
}

File: examples-sdk/csharp/403-agent-code-file-reviewer/Main.cs
// 403-agent-code-file-reviewer — C# SDK equivalent
// Compile: vil compile --from csharp --input 403-agent-code-file-reviewer/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("code-file-reviewer-agent", 3122);
var code_review_agent = new ServiceProcess("code-review-agent");
code_review_agent.Endpoint("POST", "/code-review", "code_review_handler");
server.Service(code_review_agent);
server.Compile();
